lib/_tools.js

```diff
@@ -9,19 +9,33 @@ module.exports.formatDate = function formatDate(d, dateonly, floating) {
         return (i < 10 ? '0' : '') + i;
     }
 
-    s = d.getUTCFullYear();
-    s += pad(d.getUTCMonth() + 1);
-    s += pad(d.getUTCDate());
+    // For more information about floating format, see RFC 5545, section 3.3.5
+    if(!floating) {
+      // Generate UTC Format
+      s = d.getUTCFullYear();
+      s += pad(d.getUTCMonth() + 1);
+      s += pad(d.getUTCDate());
+
+      if(!dateonly) {
+          s += 'T';
+          s += pad(d.getUTCHours());
+          s += pad(d.getUTCMinutes());
+          s += pad(d.getUTCSeconds());
+          s += 'Z';
```

Comment:
Not necessarily. If the UTC time causes a change in date from the local date (from crossing a date boundary) then it can change the day, month, or year.